src/js/components/Form/Form.js

```diff
@@ -80,6 +80,48 @@ const Form = forwardRef(
     }, [errorsProp, infosProp]);
     const validations = useRef({});
 
+    const setValid = nextErrors => {
+      let valid = false;
+
+      valid = requiredFields
+        .filter(n => Object.keys(validations.current).includes(n))
+        .every(
+          field =>
+            value[field] && (value[field] !== '' || value[field] !== false),
+        );
+
+      if (Object.keys(nextErrors).length > 0) valid = false;
+      return valid;
+    };
+
+    useEffect(() => {
+      const dirtyFields = Object.entries(validations.current).filter(
+        ([n]) => value[n] && (value[n] !== '' || value[n] !== false),
+      );
+      if (dirtyFields.length > 0) {
```

Comment:
I think we only want to do this when `validate="change"` or `validate="blur"`

Reply:
got it, done !